dashboard/src2/components/billing/mpesa/BuyPrepaidCreditsMpesa.vue

```diff
@@ -144,6 +145,14 @@ export default {
 							'Tax ID is required for payment.',
 						);
 					}
+					if(this.taxIdInput){
+						if (!pattern.test(this.taxIdInput)) {
```

Comment:
```suggestion
					if(this.taxIdInput && !pattern.test(this.taxIdInput)) {
```
This can be written as single conditional statement.

Reply:
Sure done this